examples/phi2/phi2.py

```diff
@@ -142,11 +148,16 @@ def main(raw_args=None):
     pass_flows = [[]]
     if args.finetune_method:
         pass_flows[0].append(args.finetune_method)
-        template_json["systems"]["local_system"]["config"]["accelerators"][0]["device"] = "gpu"
         # torch fine tuning does not require execution provider, just set it to CUDAExecutionProvider
-        template_json["systems"]["local_system"]["config"]["accelerators"][0]["execution_providers"] = [
-            "CUDAExecutionProvider"
-        ]
+        update_accelerator(template_json, "gpu")
+    if args.slicegpt:
```

Comment:
should we make line 153 and 161 `elif` since 149,153,161 appear to be mutually exclusive? 
@trajepl

Reply:
Please ignore. They are not mutually exclusive.